geode-core/src/test/java/org/apache/geode/test/dunit/examples/BeforeClassExampleTest.java

```diff
@@ -34,6 +34,6 @@ public static void initializeDUnit() throws Exception {
 
   @Test
   public void shouldHaveFourDUnitVMsByDefault() throws Exception {
-    assertThat(Host.getHost(0).getVMCount()).isEqualTo(4);
+    assertThat(Host.getHost(0).getVMCount()).isGreaterThan(0);
```

Comment:
This test name is now misleading since there may or may not be four VMs. Do we always have at least 4 VMs? If so then change the assert to isGreaterThan(4) and the test method name to something like `shouldHaveAtLeast4DUnitVMs`. Or, leave the assert as you have it and just change the method name to be more meaningful